openedx/core/djangoapps/user_api/urls.py

```diff
@@ -12,17 +12,22 @@
 urlpatterns = patterns(
     '',
     url(
-        r'^v1/accounts/' + USERNAME_PATTERN + '$',
+        r'^v1/accounts/%s$' % (USERNAME_PATTERN,),
```

Comment:
I'm leaning toward `{}`, because it's formatted immediately, and the name of the variable is clear enough, but if there's a convention to name all format args, I'm happy to follow it.